File: app/src/main/java/com/neome/java/api/ent/entMain/msg/MsgSpreadsheetBulkRowCommentCountGet.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.entMain.msg;

import com.neome.java.api.core.base.msg.MsgVersion;
import com.neome.java.api.meta.base.Types.RowId;

import java.util.Map;

@SuppressWarnings("unused")
public class MsgSpreadsheetBulkRowCommentCountGet extends MsgVersion
{
  public Map<RowId, String> rowIdCommentVersionMap;
}

File: app/src/main/java/com/neome/api/ent/entMain/msg/MsgSpreadsheetBulkRowCommentCountGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entMain.msg

import java.util.Map
import com.neome.api.core.base.msg.MsgVersion
import com.neome.api.meta.base.Types.RowId

interface MsgSpreadsheetBulkRowCommentCountGet : MsgVersion
{
  val rowIdCommentVersionMap: Map<RowId, String>
}